CHANGELOG

```diff
@@ -1,6 +1,9 @@
 v3.7.10 (XXXX-XX-XX)
```

Comment:
3.7.10 is already in QA, needs to be changed to 3.7.11

Reply:
Changed.